src/ui/wxWidgets/SafeCombinationPromptDlg.cpp

```diff
@@ -172,15 +175,28 @@ void SafeCombinationPromptDlg::CreateControls()
   m_scctrl = new SafeCombinationCtrl( itemDialog1, ID_PASSWORD, &m_password, wxDefaultPosition, wxDefaultSize );
   m_scctrl->SetFocus();
   flexGridSizer->Add(m_scctrl, 1, wxALL|wxEXPAND, 5);
-
-  flexGridSizer->AddStretchSpacer(0);
+// Show Combination is moved down to right side of new field "Open as read-only"
   auto showCombinationCheckBox = new wxCheckBox(this, wxID_ANY, _("Show Combination"), wxDefaultPosition, wxDefaultSize, 0 );
   showCombinationCheckBox->SetValue(false);
   showCombinationCheckBox->Bind(wxEVT_CHECKBOX, [&](wxCommandEvent& event) {m_scctrl->SecureTextfield(!event.IsChecked());});
-  flexGridSizer->Add(showCombinationCheckBox, 1, wxALL|wxEXPAND, 5);
 
   itemBoxSizer5->Add(flexGridSizer, 0, wxEXPAND|wxALL, 5);
 
+  auto *itemBoxSizer14 = new wxBoxSizer(wxHORIZONTAL);
```

Comment:
In similar dialog boxes that require a password to be entered, such checkboxes are located below the password text input field and are left-justified to it. It would be nice if the application could offer the same visual appearance to the user.

Reply:
Look and feel might be better with the tag fields aligned with the input field. I'm not sure about the needed size of the text of the two check buttons when translated, therefore moved the text left aligned to the left most place and not starting below the input field. This is a design decision I could correct, when wanted.